data-service-client/build.gradle

```diff
@@ -1,6 +1,8 @@
-description 'Blox DataService Client'
+plugins {
+    id "java"
```

Comment:
What does this syntax do differently from the `apply plugin: 'java'` syntax? Is it a functional change?